packages/gestalt/src/index.js

```diff
@@ -59,8 +60,10 @@ export {
   Collage,
   Collection,
   Column,
+  CompositeZIndex,
```

Comment:
@chrislloyd could you update the docs & provide an example of how people should use `CompositeZIndex` and `FixedZIndex`?